eng/common/scripts/Queue-Pipeline.ps1

```diff
@@ -0,0 +1,42 @@
+[CmdletBinding(SupportsShouldProcess = $true)]
+param(
+  [Parameter(Mandatory = $true)]
+  [string]$Organization,
+
+  [Parameter(Mandatory = $true)]
+  [string]$Project,
+
+  [Parameter(Mandatory = $true)]
+  [string]$SourceBranch,
+
+  [Parameter(Mandatory = $true)]
+  [int]$DefinitionId,
+
+  [Parameter(Mandatory = $true)]
+  [string]$AuthToken
```

Comment:
Do you still need me to encode the token in key-vault or did you handle that?

Reply:
I though you did that in our last meeting. But I'll check and if not I will do that on the portal